app/views/teacher_interface/application_forms/show/_awarded.html.erb

```diff
@@ -12,7 +12,7 @@
 
 <h3 class="govuk-heading-m">Find out more about teaching in England</h3>
 
-<p class="govuk-body">There’s more information about coming to teach in England, including details about visas and immigration, in the <a href="https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk" class="govuk-link">Department for Education’s guidance for teachers who qualified outside the UK</a>.</p>
+<p class="govuk-body">There’s more information about coming to teach in England at Get Into Teaching: <a href="https://getintoteaching.education.gov.uk/non-uk-teachers" class="govuk-link">https://getintoteaching.education.gov.uk/non-uk-teachers</a>.</p>
```

Comment:
This approach of including the link in the text isn't good for accessibility, it means any screen readers will read out the URL which isn't a good experience for those users. Can we instead have "Get Into Teaching" as the link text?

Reply:
I was told that we had to put the entire URL into emails - let me just check that one out...

Reply:
Apols, my error. I'm thinking of consistency across comms but happy to go back to having Get Into Teaching as the link text.

Reply:
I had that originally but changed it; I can easily put it back

Reply:
Apols @richardpattinson :)